src/modules/logger/logger.cpp

```diff
@@ -653,13 +653,13 @@ void Logger::add_default_topics()
 	add_topic("vehicle_local_position", 100);
 	add_topic("vehicle_local_position_setpoint", 100);
 	add_topic("vehicle_magnetometer", 200);
+	add_topic("vehicle_mocap_odometry");
 	add_topic("vehicle_rates_setpoint", 30);
 	add_topic("vehicle_status", 200);
 	add_topic("vehicle_status_flags");
 	add_topic("vehicle_trajectory_waypoint", 200);
 	add_topic("vehicle_trajectory_waypoint_desired", 200);
-	add_topic("vehicle_vision_attitude");
-	add_topic("vehicle_vision_position");
+	add_topic("vehicle_visual_odometry");
```

Comment:
Limit the default rates?

Reply:
Well the rates won't be above 50hz. Most of the cases will be around 30hz and bellow. Don't see a reason to limit it at those rates.

Reply:
Those messages are both 229 bytes logged. At 50 Hz that's ~ 22 kB/s, which is more than half of our current total average logging rate on a typical stm32f4 setup.

There are a lot of things I'd like to log, but we just can't without increasing dropouts on most common setups.

Please set them to something minimal in `add_default_topics()`, and add them full rate only in `add_estimator_replay_topics()`.

Reply:
@dagar done!